### PR TITLE
add duotone preset

### PR DESCRIPTION
**Pull request by LittleBigThing**
Add duotone preset to theme.json based on theme colors

Solves #140

I’d like to note that the difference between the foreground and tertiary colors is visually hardly noticeable:

duotone-presets

### CODE REVIEW

**Comment by kjellr** [comment]
Thanks for the PR, @LittleBigThing! 

> I’d like to note that the difference between the foreground and tertiary colors is visually hardly noticeable:

That's ok by me — it's subtle, but there will be use cases when one is better than the other (for example, if the image is on a tertiary-colored background, you'll want to use the tertiary-colored duotone.

File: theme.json [hunk 1]
```diff
@@ -33,6 +33,38 @@
 	],
 	"settings": {
 		"color": {
+			"duotone": [
+                {
+                    "colors": [ "#000000", "#ffffff" ],
```
Comment (kjellr) [inline theme.json:38]: Should these be defined using the CSS variables instead? Like we do for gradients? 

https://github.com/WordPress/twentytwentytwo/blob/83badf191b8922f9d825076cf49deef1b64a8c67/theme.json#L39

Or does that not work?

Comment (LittleBigThing) [reply theme.json:38]: No, it didn’t seem to work for me like this:

```
"duotone": [
                {
                    "colors": [ "var(--wp--preset--color--foreground)", "var(--wp--preset--color--background)" ],
                    "slug": "foreground-and-background",
                    "name": "Foreground and background"
                }
```

Maybe I'm doing something wrong?
The palette is defined in hex codes as well.

Comment (kjellr) [reply theme.json:38]: Yeah, it looks like it's not possible at this point. 👍

**Review by kjellr — APPROVED** [review]
Looks good, thanks @LittleBigThing!

**Pull request merged**